rules/ktlint/src/test/kotlin/com/twitter/compose/rules/ktlint/ComposeNamingCheckTest.kt

```diff
@@ -23,6 +24,21 @@ class ComposeNamingCheckTest {
         namingRuleAssertThat(code).hasNoLintViolations()
     }
 
+    @Test
+    fun `passes when a composable that returns values is uppercase but allowed`() {
```

Comment:
this test seems to be failing

Reply:
Fixed! I misunderstood how the naming worked. It does make the detekt name a bit wordy in context, but not sure there's an alternative to support both easily.